ts/IdentificationModal.tsx

```diff
@@ -234,14 +262,26 @@ class IdentificationModal extends React.PureComponent<Props, State> {
   };
 
   private onIdentificationFailureHandler = () => {
-    const { dispatch } = this.props;
-    dispatch(identificationFailure());
+    const { dispatch, identificationFailState } = this.props;
+
+    const forceLogout = fromNullable(identificationFailState)
```

Comment:
Since this Component won't be never unmount, you should clear the interval check before dispatch identificationForceLogout